packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -185,6 +185,11 @@ export default class VTable {
      * @param operation Table operation
      */
     edit(operation: TableOperation | CompatibleTableOperation) {
+        if (this.selection && (this.row === undefined || this.col == undefined)) {
+            this.col = 0;
+            this.row = 0;
+        }

```

Comment:
hmm, so we sometimes read col and row, sometimes read selection?

I think we should always read selection, and deprecate col and row